### PR TITLE
Add buttons to psbt dialog

### PR DESCRIPTION
**Pull request by giacomocaironi**
Add buttons to import/export a psbt from/to a file

### CODE REVIEW

**Comment by achow101** [comment]
Files containing PSBTs can be both text files that contain the PSBT in base64, or the binary data of the PSBT itself. So "Open file" needs to be able to handle both, and "Save file" should present an option to the user to choose which type to create. For saving, I think we should default to saving as a binary file with a `.psbt` extension as the BIP specifies.

**Comment by giacomocaironi** [comment]
With the new commit the user has the option to decide in which format to load/save a transaction. So there is not a default and the user has to specify the format each time. Is that ok?

File: hwilib/_gui.py [hunk 1]
```diff
@@ -304,7 +351,7 @@ def __init__(self, passphrase='', chain=Chain.MAIN):
 
     def clear_info(self):
         self.ui.getxpub_button.setEnabled(False)
-        self.ui.signtx_button.setEnabled(False)
+        self.ui.signtx_button.setEnabled(True)
```
Comment (achow101) [inline hwilib/_gui.py:354]: I don't think this is correct.

Comment (giacomocaironi) [reply hwilib/_gui.py]: Sorry, I missed it. Now it is fixed

**Comment by achow101** [comment]
ACK a98ac31a4f4b67a4fcdcac33f42ccd3ed97b3a62

**Pull request merged**